docker-compose.yml

```diff
@@ -1,5 +1,40 @@
 version: '2'
 services:
+  antispam:
+    image: mailu/rspamd:stable
+    restart: always
+    volumes:
+      - /var/lib/rspamd
+
+  antivirus:
+    image: mailu/clamav:stable
+    restart: always
+    volumes:
+      - /data
+
+  certbot:
+    build: certbot
+    restart: always
+    env_file: env
```

Comment:
I'd suggest to leave this line out, and put the relevant environment variables in `environment` instead - if they are really local, we can just not give them a value, so they are taken from the environment. `.env` is still recognized.

Reply:
it's a copy&past from my setup, where `env_file` makes more sense. here is no purpose for it